var/spack/repos/builtin/packages/librom/package.py

```diff
@@ -0,0 +1,74 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import glob
+
+class Librom(AutotoolsPackage):
+    """libROM: library for computing large-scale reduced order models"""
+
+    homepage = "https://github.com/LLNL/libROM"
+    url      = "https://github.com/LLNL/libROM"
+
+    version('develop', git='https://github.com/LLNL/libROM',
+            branch='master')
+
+    depends_on('lapack')
+    depends_on('mpi')
+    depends_on('zlib')
+    depends_on('libszip')
+    depends_on('hdf5')
+    depends_on('perl')
+    depends_on('graphviz')
+    depends_on('doxygen')
+    depends_on('boost')
+
+    def configure_args(self):
+        spec = self.spec
+        args = ['--with-lapack={0}'.format(spec['lapack'].prefix),
```

Comment:
Can u specify libs or link flags instead of prefix? 

Otherwise Spack’s preferences for threading and 64Bit indexes will likely be ignored

Reply:
The new commit specifies both. Specifying the link flags will override the defaults and should preserve threading and 64-bit indexing, while also specifying the includes directories in case C interfaces to BLAS/LAPACK are used.